### PR TITLE
Hello, here is the 13icons you asked on twitter 

### PR DESCRIPTION
**Pull request by Virgile-fr**
Hello Gil,

I hope this message finds you well. Following [your tweet](https://twitter.com/gilgNYC/status/1753165604695994686) seeking a designer for shipping carrier icons that match Active Merchant's payment icons, I'm excited to let you know that I've completed the design work and believe I've created icons that fit the bill.

Hope it's working for you :)

### CODE REVIEW

File: db/carrier_icons.yml [hunk 1]
```diff
@@ -2,11 +2,52 @@
...
   name: fedex
-  label: FedEx
+  label: Fedex
+  group: other
+-
+  name: evri
+  label: Evri
+  group: other
+-
+  name: dpd
```
Comment (gil--) [inline db/carrier_icons.yml]: For DPD, I think it has to be horizontal per guidelines: https://www.dpd.com/cz/wp-content/uploads/sites/226/2020/05/DPD_Logo-manu%C3%A1l.pdf
CleanShot 2024-02-02 at 12 23 15@2x

CleanShot 2024-02-02 at 12 23 35@2x

Any thoughts on providing both the icon and then one with the symbol+logo. maybe dpd_symbol.svg
CleanShot 2024-02-02 at 12 26 13@2x

File: db/carrier_icons.yml [hunk 2]
```diff
@@ -2,11 +2,52 @@
...
+-
+  name: correos
+  label: Correos
+  group: other
+-
+  name: colissimo
+  label: Colissimo
+  group: other
+-
+  name: chronopost
```
Comment (gil--) [inline db/carrier_icons.yml]: CleanShot 2024-02-02 at 12 26 49@2x

Like DPD, we can provide a chronopost_symbol.svg with just the chronopost blue box. https://www.dpd.com/cz/wp-content/uploads/sites/226/2020/05/DPD_Logo-manu%C3%A1l.pdf

File: db/carrier_icons.yml [hunk 3]
```diff
@@ -2,11 +2,52 @@
   name: ups
```
Comment (gil--) [inline db/carrier_icons.yml]: Per UPS brand guidelines, can we place the logo on white background? 

CleanShot 2024-02-02 at 12 30 11@2x

File: db/carrier_icons.yml [hunk 4]
```diff
@@ -2,11 +2,52 @@
...
   label: UPS
   group: other
+-
```
Comment (gil--) [inline db/carrier_icons.yml:5]: Can you supply a usps_symbol.svg variant? Just the symbol without "USPS" text

CleanShot 2024-02-02 at 12 29 13@2x

File: db/carrier_icons.yml [hunk 5]
```diff
@@ -2,11 +2,52 @@
...
 -
   name: dhl
   label: DHL
   group: other
+-
+  name: correos
+  label: Correos
+  group: other
+-
+  name: colissimo
```
Comment (gil--) [inline db/carrier_icons.yml]: Could you create a colissimo_symbol.svg version: https://www.colissimo.entreprise.laposte.fr/en/tools-and-services/communication-kit

CleanShot 2024-02-02 at 12 35 35@2x

Comment (gil--) [reply db/carrier_icons.yml]: CleanShot 2024-02-02 at 12 39 54@2x

File: db/carrier_icons.yml [hunk 6]
```diff
@@ -2,11 +2,52 @@
...
+  label: PosteItaliane
+  group: other
+-
+  name: mondial-relay
+  label: MondialRelay
+  group: other
 -
   name: fedex
-  label: FedEx
+  label: Fedex
```
Comment (gil--) [inline db/carrier_icons.yml]: Can we remove "Express" and just have FedEx logo? Adding Express can limit uses when it mig
CleanShot 2024-02-02 at 12 39 15@2x
ht be ground/overnight/freight.

File: db/carrier_icons.yml [hunk 7]
```diff
@@ -2,11 +2,52 @@
...
+  name: sendle
+  label: Sendle
+  group: other
+  
+-
+  name: poste-italiane
+  label: PosteItaliane
+  group: other
+-
+  name: mondial-relay
```
Comment (gil--) [inline db/carrier_icons.yml]: Can we make sure to use the red logo on white background per https://www.mondialrelay.fr/solutionspro/documentation-technique/kit-de-communication/

CleanShot 2024-02-02 at 12 41 00@2x

If you're up for it, include a mondial-relay_symbol.svg variant of just the symbol:


CleanShot 2024-02-02 at 12 42 04@2x

File: db/carrier_icons.yml [hunk 8]
```diff
@@ -2,11 +2,52 @@
...
+-
+  name: dpd
+  label: Dpd
   group: other
 -
   name: dhl
   label: DHL
   group: other
+-
+  name: correos
```
Comment (gil--) [inline db/carrier_icons.yml]: Per Correos new image guidelines, can you only include the symbol?

https://www.correos.com/en/sala-prensa/#imagen-marca

Alternatively, we could just add a correos_symbol.svg variant?

CleanShot 2024-02-02 at 12 43 53@2x

File: db/carrier_icons.yml [hunk 9]
```diff
@@ -2,11 +2,52 @@
...
+-
+  name: colissimo
+  label: Colissimo
+  group: other
+-
+  name: chronopost
+  label: Chronopost
+  group: other
+-
+  name: canadapost
```
Comment (gil--) [inline db/carrier_icons.yml]: Would you be able to add a canadapost_symbol.svg variant?

https://www.canadapost-postescanada.ca/cpc/en/our-company/news-and-media/media-centre/canada-post-logos.page

CleanShot 2024-02-02 at 12 47 00@2x

**Review by gil-- — CHANGES_REQUESTED** [review]
Thank you @Virgile-fr so much for helping with this! Have some requested changes to ensure we more closely match official brand guidelines.

Send a DM to https://twitter.com/gilgNYC with your Paypal email and after merging, I'll send $20/icon to your Paypal.

**Comment by Virgile-fr** [comment]
Hi  @gil--  ,

Thank you for your feedback. No problem with the changes; I'm on it. I'll update you as soon as possible.

**Comment by Virgile-fr** [comment]
Hey @gil--  , Just to keep you updated, all the requested changes have been made. I've included the variants, corrected the logos, and the YML file is up to date.

There are now 20 SVGs in total, considering the additions 

I hope this matches what you were looking for. Once you've had a chance to check everything, I'll be ready to share my PayPal details through Twitter DM if all is well.

Best wishes.

**Review by gil-- — APPROVED** [review]
Thanks for implementing the changes and the symbols!
I think this is a great v1 and will hopefully be useful for the wider community.

**Comment by gil--** [comment]
@Virgile-fr Send me a DM on Twitter when you have time.

**Comment by Virgile-fr** [comment]
Hi @gil--  , really happy that everything is working :) , I sent you a DM on @gilgNYC from my Twitter account (@irgile), but I think the messages are being marked as spam... I'll send you another DM from another account (@sidfik). Let me know if you receive it properly. :)

If not, try to send me a message at this link: https://twitter.com/irgile. There should be a request for messages. :)

**Pull request merged**